### PR TITLE
Test API routes

### PR DESCRIPTION
**Pull request by jacetchan**
If merged, this pull request will:
- [Create unit tests for API routes](https://trello.com/c/nw4YPGYm)

### CODE REVIEW

File: tests/index.test.js [hunk 1]
```diff
@@ -1,6 +1,201 @@
...
+describe('Test GET /reviews/listingId/1', () => {
+
+  test('It should respond with a status code', (done) => {
+    request(app).get('/reviews/listingId/1').then((response) => {
+      expect(response.statusCode).toBe(200);
+      done();
+    });
+  });
+
+  test('It should send aggregate over ratings for 1 listing', (done) => {
```
Comment (LYRADNIS) [inline tests/index.test.js]: The wording of the test is a bit confusing!

File: tests/index.test.js [hunk 2]
```diff
@@ -1,6 +1,201 @@
 const request = require('supertest');
 const app = require('../server/app');
 
-test('adds 3 to equal 3', () => {
-  expect(3).toBe(3);
+describe('Test GET /', () => {
+
+  test('It should respond with a status code', (done) => {
```
Comment (LYRADNIS) [inline tests/index.test.js]: Nit picking here as I'm guilty of doing this also. lol
Using 'It' might be a bit vague. Maybe '"Get" request should respond with ...' 
But other than that looks great!

File: tests/app.test.js [hunk 3]
```diff
@@ -283,4 +283,22 @@ describe('Test GET /reviews/listingId/2/reviews', () => {
       done();
     });
   });
+});
+
+describe('Test GET /reviews/listingId/:listingId', () => {
+  test('It should respond with a 500 status code', (done) => {
```
Comment (LYRADNIS) [inline tests/app.test.js]: Again nitpicking here; But 'it' is a bit ambiguous.

**Comment by jacetchan** [comment]
Thanks for pointing out the intent of Line 30. Just pushed a fix for it! :)

For the usage of 'It', I was referencing the guide Natalie linked us to:
- [How to test Express.js with Jest and Supertest](http://www.albertgao.xyz/2017/05/24/how-to-test-expressjs-with-jest-and-supertest/)

**Pull request merged**